packages/frontend/src/singleMovie/components/VideoHeaderComponent.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import { fontFamily } from "../../utils/fontFamily";
+import { fontSize } from "../../utils/fontSize";
+import { fontWeight } from "../../utils/fontWeight";
+import { image } from "../../utils/image";
+import { Colors } from "../../utils/colors";
+import { breakPoint } from "../../utils/breakPoint";
+
+const Header = styled.header`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 20px;
+`;
+
+const Title = styled.h1`
+  font-family: ${fontFamily.openSansFont};
+  font-size: ${fontSize.headerMobile};
+  font-weight: ${fontWeight.black};
+  color: ${Colors.colorMainFont};
+  @media screen and (min-width: ${breakPoint.desktop}) {
+    font-size: ${fontSize.headerDesktop};
+  }
+`;
+
+const VideoRating = styled.div`
+  line-height: 40px;
+  padding-right: 15px;
+  display: flex;
+  align-content: center;
+`;
+
+const VideoRatingValue = styled.div`
+  font-family: ${fontFamily.openSansFont};
+  font-size: ${fontSize.subtitleMobile};
+  font-weight: ${fontWeight.black};
+  line-height: 40px;
+  padding-left: 10px;
+  display: flex;
+  align-content: center;
+  color: ${Colors.colorMainFont};
```

Comment:
a mozesz zmienić kolorowi nazwe tak zeby było wiadomo co to za kolor

Reply:
to nie mój kod był już napisany